src/EditorFeatures/CSharpTest/BlockCommentEditing/BlockCommentEditingTests.cs

```diff
@@ -0,0 +1,563 @@
+$$*
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine0()
+        {
+            var code = @"
+    /*
+     *$$
+";
+            var expected = @"
+    /*
+     *
+     * $$
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine1()
+        {
+            var code = @"
+    /*
+     *$$*/
+";
+            var expected = @"
+    /*
+     *
+     $$*/
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine2()
+        {
+            var code = @"
+    /*
+     *$$ */
+";
+            var expected = @"
+    /*
+     *
+     *$$ */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine3()
+        {
+            var code = @"
+    /*
+     * $$ 1.
+     */
+";
+            var expected = @"
+    /*
+     * 
+     * $$ 1.
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine4()
+        {
+            var code = @"
+    /*
+     *  1.$$
+     */
+";
+            var expected = @"
+    /*
+     *  1.
+     *  $$
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine5()
+        {
+            var code = @"
+    /*
+     *   1.
+     *   $$
+     */
+";
+            var expected = @"
+    /*
+     *   1.
+     *   
+     *   $$
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine6()
+        {
+            var code = @"
+    /*
+  $$   *
+     */
+";
+            var expected = @"
+    /*
+  
+     * $$   *
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine7()
+        {
+            var code = @"
+    /*
+     *************$$
+     */
+";
+            var expected = @"
+    /*
+     *************
+     * $$
+     */
+";
+            await VerifyAsync(code, expected);
+        }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BlockCommentEditing)]
+        public async Task InsertOnMiddleLine8()
+        {
+            var code = @"
+    /**
+     *$$
+     */
+";
+            var expected = @"
+    /**
+     *
+     * $$
```

Comment:
Same as VS2013. I think it's padding instead of indent. It's uncommon to get into this case. But as soon as people find out this feature does not clean up trailing spaces, they could probably want to <kbd>Backspace</kbd> that padding before hitting <kbd>Enter</kbd> to make a blank line.